modules/clightning-plugins/feeadjuster.nix

```diff
@@ -0,0 +1,60 @@
+{ config, lib, ... }:
+
+with lib;
+let cfg = config.services.clightning.plugins.feeadjuster; in
+{
+  options.services.clightning.plugins.feeadjuster = {
+    enable = mkEnableOption "Feeaduster (clightning plugin)";
+    fuzz = mkOption {
+      type = types.bool;
+      default = true;
+      description = "Enable update threshold randomization and hysterisis";
+    };
+    adjustOnForward = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Automatically update fees on forward events";
+    };
+    method = mkOption {
+      type = types.enum [ "soft" "default" "hard" ];
+      default = "default";
+      description = "Adjustment method to calculate channel fee (soft=less difference, hard=high difference)";
+    };
+    adjustDaily = mkOption {
+      type = types.bool;
+      default = true;
+      description = "Automatically update fees daily";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    services.clightning.extraConfig = ''
+      plugin=${config.nix-bitcoin.pkgs.clightning-plugins.feeadjuster.path}
+      feeadjuster-adjustment-method="${cfg.method}"
+    '' +
+    (optionalString (!cfg.fuzz) "feeadjuster-deactivate-fuzz") +
```

Comment:
This is missing a trailing newline, so all options are defined in a single line without separation. This is all worked out in my fixup.

Reply:
I used your version